app/src/main/java/ch/hikemate/app/ui/auth/SignInWithEmailScreen.kt

```diff
@@ -0,0 +1,148 @@
+  const val TEST_TAG_PASSWORD_INPUT = "sign_in_with_email_password_input"
+  const val TEST_TAG_SIGN_IN_BUTTON = "sign_in_with_email_sign_in_button"
+  const val TEST_TAG_GO_TO_SIGN_UP_BUTTON = "sign_in_with_email_go_to_sign_up_button"
+}
+
+/**
+ * A composable that displays the sign in with email screen.
+ *
+ * @param navigationActions The navigation actions.
+ * @param authViewModel The authentication view model.
+ */
+@Composable
+fun SignInWithEmailScreen(navigationActions: NavigationActions, authViewModel: AuthViewModel) {
+  val context = LocalContext.current
+
+  // Define the colors for the input fields
+  val inputColors =
+      OutlinedTextFieldDefaults.colors()
+          .copy(
+              focusedLabelColor = primaryColor,
+              focusedIndicatorColor = primaryColor,
+              cursorColor = primaryColor,
+              textSelectionColors =
+                  TextSelectionColors(
+                      handleColor = primaryColor,
+                      backgroundColor = primaryColor,
+                  ))
+
+  // Define the email and password state
+  var email by remember { mutableStateOf("") }
+  var password by remember { mutableStateOf("") }
+
+  Column(
+      modifier =
+          Modifier.testTag(Screen.SIGN_IN_WITH_EMAIL)
+              .padding(
+                  // Add padding to the sidebar padding
+                  start = 16.dp,
+                  end = 16.dp,
+                  top = 16.dp,
+              ),
+      verticalArrangement = Arrangement.spacedBy(16.dp)) {
+        BackButton(navigationActions)
+        Text(
+            context.getString(R.string.SIGN_IN_WITH_EMAIL_TITLE),
+            style = TextStyle(fontWeight = FontWeight.Bold, fontSize = 32.sp),
+            modifier = Modifier.testTag(SignInWithEmailScreen.TEST_TAG_TITLE))
+
+        OutlinedTextField(
+            modifier = Modifier.fillMaxWidth().testTag(SignInWithEmailScreen.TEST_TAG_EMAIL_INPUT),
+            keyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Email),
+            colors = inputColors,
+            value = email,
+            onValueChange = { email = it },
+            label = { Text(context.getString(R.string.SIGN_IN_WITH_EMAIL_EMAIL_LABEL)) })
+
+        OutlinedTextField(
+            modifier =
+                Modifier.fillMaxWidth().testTag(SignInWithEmailScreen.TEST_TAG_PASSWORD_INPUT),
+            visualTransformation = PasswordVisualTransformation(),
+            colors = inputColors,
+            value = password,
+            onValueChange = { password = it },
+            label = { Text(context.getString(R.string.SIGN_IN_WITH_EMAIL_PASSWORD_LABEL)) })
+
+        BigButton(
+            modifier =
+                Modifier.fillMaxWidth().testTag(SignInWithEmailScreen.TEST_TAG_SIGN_IN_BUTTON),
+            buttonType = ButtonType.PRIMARY,
+            label = context.getString(R.string.SIGN_IN_WITH_EMAIL_SIGN_IN_BUTTON),
```

Comment:
Nitpick: use the composable function `stringResource` instead.

```suggestion
            label = stringResource(R.string.SIGN_IN_WITH_EMAIL_SIGN_IN_BUTTON),
```